scalardb/README.md

```diff
@@ -0,0 +1,59 @@
+# Jepsen tests for Scalar DB
```

Comment:
This needs to be updated I think.

Reply:
Updated

Reply:
Maybe for number 1 you can say something like:

1. Start the Docker nodes and log into jepsen-control (as explained [here](../README.md)).